### PR TITLE
sciarg relations hparam tuning config

### PR DESCRIPTION
**Pull request by jk-tripathy**

### CODE REVIEW

File: configs/hparams_search/sciarg_relations.yaml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      _target_: optuna.samplers.RandomSampler
+      seed: 12345
+
+    # define range of hyperparameters
+    # More information here : https://hydra.cc/docs/plugins/optuna_sweeper/#search-space-configuration
+    params:
+      +trainer.gradient_clip_val: choice(0.5, 0.6, 0.7, 0.8, 0.9)
+      model.learning_rate: choice(5e-6, 7e-6, 3e-5, 5e-5, 7e-5, 3e-4, 5e-4)
+      +model.task_learning_rate: choice(1e-5, 3e-5, 7e-5, 1e-4, 3e-4, 5e-4, 7e-4)
+      model.warmup_proportion: choice(0.0, 0.1, 0.2, 0.3)
```
Comment (ArneBinder) [inline configs/hparams_search/sciarg_relations.yaml]: please change `warmup_proportion` as dicussed

**Review by ArneBinder — CHANGES_REQUESTED** [review]
see below

File: configs/hparams_search/sciarg_relations.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    # you can choose bayesian sampler (tpe), random search (without optimization), grid sampler, and others
+    # docs: https://optuna.readthedocs.io/en/stable/reference/samplers.html
+    sampler:
+      _target_: optuna.samplers.RandomSampler
+      seed: 12345
+
+    # define range of hyperparameters
+    # More information here : https://hydra.cc/docs/plugins/optuna_sweeper/#search-space-configuration
+    params:
+      +trainer.gradient_clip_val: choice(0.5, 0.6, 0.7, 0.8, 0.9)
```
Comment (ArneBinder) [inline configs/hparams_search/sciarg_relations.yaml:54]: what motivated the choice of these values? is this fine to use for gradient clipping? e.g. what the "maximum range" for that? and then choose accordingly

**Review by ArneBinder — CHANGES_REQUESTED** [review]
one thing

**Review by ArneBinder — APPROVED** [review]
